src/main/java/com/butbetter/applicationservices/externalAPI/service/TranslatorService.java

```diff
@@ -0,0 +1,25 @@
+package com.butbetter.applicationservices.externalAPI.service;
+
+import com.butbetter.applicationservices.externalAPI.model.Language;
+
+public interface TranslatorService {
+
+    /**
+     * returns the language of a given text
+     * @param text of which you want to know the language
+     *             make sure not to exceed The request body size of 128 KiB (128 * 1024 bytes)
+     * @return the language as a enum Value
+     */
+    Language getLanguage(String text);
+
+    /**
+     * returns the translation of the GERMAN!!! text into the defined target language
```

Comment:
German only i see erst recht mit den Enum 🗡️

Reply:
Was meinst du hiermit?

Reply:
Das gibt doch Jede Translation zurück nicht nur Deutsch

Reply:
ja steht da ja auch: Übersetzung eines deutschen Textes in die von DeepL unterstützten Sprachen
aber du kannst z.B. keinen englischen Text ins Deutsche übersetzen mit der Methode